code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -68,9 +70,16 @@ GLOBAL_VAR(bomb_set)
 	. = ..()
 	r_code = rand(10000, 99999) // Creates a random code upon object spawn.
 	wires = new/datum/wires/nuclearbomb(src)
-	ADD_TRAIT(src, TRAIT_OBSCURED_WIRES, ROUNDSTART_TRAIT)
+	if(is_syndicate)
+		wires.labelled = FALSE
+		ADD_TRAIT(src, TRAIT_OBSCURED_WIRES, ROUNDSTART_TRAIT)
 	previous_level = SSsecurity_level.get_current_level_as_text()
-	GLOB.poi_list |= src
+	if(!training)
+		GLOB.poi_list |= src
+		if(is_syndicate)
+			GLOB.syndi_nuke_list |= src
+		else
+			GLOB.nt_nuke_list |= src
```

Comment:
This should be done oop way, so redefining stuff in subclasses. Eg adding a proc add_to_nuke_list which ll add to nt list and redefine it in syndie bomb. Same with labeling and training

Reply:
Adressed, moved stuff to initialize subprocs